src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -1,29 +1,49 @@
 {#
-
-This file is part of the Sonata package.
-
-(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
-
-For the full copyright and license information, please view the LICENSE
-file that was distributed with this source code.
-
-#}
-
+ 
+ This file is part of the Sonata package.
+ 
+ (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ 
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+ 
+ #}
 {% block sonata_security_roles_widget %}
-{% spaceless %}
-    <div class="editable">
-        <h4>{{ 'field.label_roles_editable'|trans({}, "SonataUserBundle") }}</h4>
-        {{ block('choice_widget') }}
-    </div>
-    {% if read_only_choices|length > 0 %}
-    <div class="readonly">
-        <h4>{{ 'field.label_roles_readonly'|trans({}, "SonataUserBundle") }}</h4>
-        <ul>
-        {% for choice in read_only_choices %}
-            <li>{{ choice }}</li>
-        {% endfor %}
-        </ul>
-    </div>
-    {% endif %}
-{% endspaceless %}
+    {% spaceless %}
+        <div class="editable">
+            <h4>{{ 'field.label_roles_editable'|trans({}, "SonataUserBundle") }}</h4>
+            {{ block('choice_widget') }}
+        </div>
+        {% if read_only_choices|length > 0 %}
+            <div class="readonly">
+                <h4>{{ 'field.label_roles_readonly'|trans({}, "SonataUserBundle") }}</h4>
+                <ul>
+                    {% for choice in read_only_choices %}
+                        <li>{{ choice }}</li>
+                    {% endfor %}
+                </ul>
+            </div>
+        {% endif %}
+    {% endspaceless %}
 {% endblock sonata_security_roles_widget %}
+
+{% block sonata_roles_matrix_widget %}
+    {% spaceless %}
+        <table class="table table-condensed">
+            <thead>
+            <tr>
+                <th></th>
+                {% for label in label_permission %}
+                    <th> {{ label }} </th>
+                {% endfor %}
+            </tr>
+            </thead>
+            <tbody>
+            {{ renderTable(form)|raw }}
```

Comment:
as we are avoiding the word `table` in favor of `matrix` this should be `renderMatrix`

cc @kunicmarko20